variables.tf

```diff
@@ -82,6 +82,18 @@ variable "kms_data_key_reuse_period_seconds" {
   default     = 300
 }
 
+variable "deduplication_scope" {
+  description = "Specifies whether message deduplication occurs at the message group or queue level"
+  type        = string
+  default     = "queue"
+}
+
+variable "fifo_throughput_limit" {
```

Comment:
Make sure that you also update `main.tf` and use these variables there as arguments:

https://github.com/terraform-aws-modules/terraform-aws-sqs/blob/master/main.tf#L1-L21

Reply:
fixed 🤦‍♂️